commander/test/bootstrapping/commands/generator.spec.ts

```diff
@@ -36,54 +35,101 @@ describe('forging', () => {
 		jest.spyOn(process.stdout, 'write').mockImplementation(val => stdout.push(val as string) > -1);
 		jest.spyOn(process.stderr, 'write').mockImplementation(val => stderr.push(val as string) > -1);
 		jest.spyOn(appUtils, 'isApplicationRunning').mockReturnValue(true);
-		invokeMock = jest.fn().mockResolvedValue({ address: 'actionAddress', forging: true });
+		invokeMock = jest.fn();
 		jest.spyOn(apiClient, 'createIPCClient').mockResolvedValue({
 			disconnect: jest.fn(),
 			invoke: invokeMock,
 		} as never);
 		jest.spyOn(inquirer, 'prompt').mockResolvedValue({ password: 'promptPassword' });
 	});
 
-	describe('forging:enable', () => {
+	describe('generator:enable', () => {
 		it('should throw an error when arg is not provided', async () => {
-			await expect(EnableCommand.run([], config)).rejects.toThrow('Missing 4 required arg');
+			await expect(EnableCommand.run([], config)).rejects.toThrow('Missing 1 required arg');
 		});
 
 		it('should throw an error when height, maxHeightGenerated and maxHeightPrevoted arg is not provided', async () => {
 			await expect(
 				EnableCommand.run(['myAddress', '--password=my-password'], config),
-			).rejects.toThrow('Missing 3 required arg');
+			).rejects.toThrow(
+				'The height, max-height-generated and max-height-prevoted values must be greater than or equal to 0',
```

Comment:
Now that we use kebab-case for flags in error message, it would be good to make it consistent also in the name of this test case and the following test case. And also to use plural tense:

```
should throw an error when height, max-height-generated and max-height-prevoted values are not provided
```